documentation/domains/Domain.md

```diff
@@ -190,6 +191,8 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 | `desiredState` | string | Desired state of this WebLogic Server instance. Values are RUNNING, ADMIN, or SHUTDOWN. |
 | `health` | [Server Health](#server-health) | Current status and health of a specific WebLogic Server instance. |
 | `nodeName` | string | Name of Node that is hosting the Pod containing this WebLogic Server instance. |
+| `podPhase` | string | Phase of server pod. Possible values are: Pending, Succeeded, Failed, Running, or Unknown. |
+| `podReady` | string | Status of server pod's condition type Ready if the pod is in Running phase, otherwise Unknown. Possible values are: True, False or Unknown. |
```

Comment:
Status of server pod's condition type Ready if the pod is in Running phase, otherwise Unknown. -> (I'm having a hard time interpreting this description, perhaps, due to a lack of punctuation.) -> Do you mean, "Status of the server pod's condition type; Ready if the pod is in the Running phase, otherwise Unknown."

Reply:
Ready is a pod condition type.

Reply:
how about "Status of server pod's Ready condition if the pod is in Running phase, otherwise Unknown."?

Reply:
@doxiao So, then does it make sense to write it as I suggested? "Status of the server pod's condition type; (semi-colon)  Ready if the pod is in the Running phase, otherwise Unknown."

Reply:
@doxiao oh, now I understand. Yes, your suggestion is ok. "Status of the server pod's Ready condition if the pod is in Running phase, otherwise Unknown."

Reply:
fixed. thanks